compat/package.json

```
@@ -34,19 +34,44 @@
       "import": "./server.mjs",
       "require": "./server.js"
     },
+    "./jsx-runtime": {
+      "import": "./jsx-runtime.mjs",
+      "require": "./jsx-runtime.js"
+    },
+    "./jsx-dev-runtime": {
+      "import": "./jsx-dev-runtime.mjs",
+      "require": "./jsx-dev-runtime.js"
+    },
+    "./scheduler": {
+      "import": "./scheduler.mjs",
+      "require": "./scheduler.js"
+    },
     "./react-19": {
+      "types": "./src/index.d.ts",
```

Oh this should be compat

```suggestion
      "types": "./compat/src/index.d.ts",
```